src/AspNet.Security.OAuth.Bitrix24/Claims.md

```diff
@@ -0,0 +1,37 @@
+﻿Available claims
```

Comment:
This should be part of a `bitrix24.md` file in the `docs` folder folding the format of the others, with some explanatory text for the claims.